moveit_core/robot_model/CMakeLists.txt

```diff
@@ -12,20 +22,24 @@ add_library(${MOVEIT_LIB_NAME}
   src/revolute_joint_model.cpp
   src/robot_model.cpp
   )
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+# set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
```

Comment:
Please add a comment about why this is commented out... e.g. TODO Fix versioning in moveit

Reply:
This should simply not be commented out according to the current Moveit release policy.
I commented on this in another request too.

Reply:
db9f036179f24131217477fc0da50b5cb35a0263